app/code/Transoft/Blog/Ui/Component/Listing/Column/Thumbnail.php

```diff
@@ -0,0 +1,82 @@
+<?php
+declare(strict_types=1);
+namespace Transoft\Blog\Ui\Component\Listing\Column;
+
+use Magento\Framework\ObjectManagerInterface;
+use Magento\Framework\UrlInterface;
+use Magento\Framework\View\Element\UiComponent\ContextInterface;
+use Magento\Framework\View\Element\UiComponentFactory;
+use Magento\Ui\Component\Listing\Columns\Column;
+
+/**
+ * Class Thumbnail
+ *
+ * UI component thumbnail class
+ */
+class Thumbnail extends Column
+{
+    const NAME = 'thumbnail';
+
+    const ALT_FIELD = 'name';
+
+    private $_objectManager = null;
+    /**
+     * @var UrlInterface
+     */
+    private $urlBuilder;
```

Comment:
Remove field. Use \Magento\Framework\View\Element\UiComponent\ContextInterface::getUrl

Reply:
\Magento\Framework\View\Element\UiComponent\ContextInterface method 'getUrl' returns url including 
parameters(which are not expected here), i couldn't find a way to get only base url like 'http:/mysite.local' using this method. 
Urlinterface is now used in 'app/code/Transoft/Blog/ViewModel/Bloglist.php'. Is there a difference between using
this interface within Thumbnail.php and Bloglist.php?

Reply:
See comment in the previous file. Please use context::getUrl, where context is defined

Reply:
fixed

Reply:
I are still using urlBuilder

Reply:
Magento\Framework\View\Element\UiComponent\ContextInterface doesn't have methods to return BaseUrl. Method getUrl return only parameters which is not compatible with this case.

Reply:
Magento\Framework\View\Element\UiComponent\ContextInterface::getUrl('', ['_direct' => 'image_path']);
Please refer to \Magento\Framework\Url::getRouteUrl

Reply:
thanks for the tip, now fixed